torch/_inductor/codecache.py

```diff
@@ -108,7 +108,11 @@ def _compile_end() -> None:
 def cache_dir() -> str:
     cache_dir = os.environ.get("TORCHINDUCTOR_CACHE_DIR")
     if cache_dir is None:
-        cache_dir = f"{tempfile.gettempdir()}/torchinductor_{getpass.getuser()}"
+        cache_dir = (
+            f"{tempfile.gettempdir()}/torchinductor_{getpass.getuser()}".replace(
```

Comment:
Would you mind doing the \ replacement only on the user string? And since we're changing this code anyway, would you mind switching it to use os.path.join()?

Reply:
ok